testing/testing.go

```diff
@@ -8,7 +8,11 @@ import (
 
 	"github.com/spf13/viper"
 
-	utils "github.com/nano-interactive/go-utils"
+	"github.com/nano-interactive/go-utils"
+)
+
+var (
+	ErrConfigNotFound = errors.New("config file not found")
```

Comment:
remove this line and inline it as this code is only for testing

Reply:
Resolved.